__tests__/app.test.js

```diff
@@ -3,24 +3,39 @@ const testData = require("../db/data/test-data");
 const { app } = require("../app");
 const seed = require("../db/seeds/seed");
 const db = require("../db/connection");
+
 beforeEach(() => seed(testData));
+afterAll(() => {
+  db.end();
+});
 
-describe.only("API Testing", () => {
-    describe("Ticket 3: GET /api/topics", () => {
-      test("returns an array of topics", () => {
-        return request(app)
-          .get("/api/topics")
-          .expect(200)
-          .then(({ body }) => {
-            body.topics.forEach((topic) => {
-              expect(topic).toEqual(
-                expect.objectContaining({
-                    slug: expect.any(String),
-                    description: expect.any(String)
-                })
-              );
-            });
+describe("API Testing", () => {
+  describe("Ticket 3: GET /api/topics", () => {
+    test("returns an array of topics", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          body.topics.forEach((topic) => {
+            expect(topic).toEqual(
+              expect.objectContaining({
+                slug: expect.any(String),
+                description: expect.any(String),
+              })
+            );
           });
-      });
+        });
+    });
+  });
+
+  describe.only("Ticket 4: GET /api/articles", () => {
+    test("Error handling", () => {
+      return request(app)
+        .get("/api/articles")
```

Comment:
I'd expect this path to return a status 200, no? It's a legitimate path in your app.js, isn't it?